### DISCUSSION TITLE
Option to show out-of-stock products at the end of the products list

### DISCUSSION THREAD

**Discussion by mkarim1378** [elementor/elementor#25122] [Feature Request]
### Prerequisites

- [x] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Currently, when designing product listings within **Elementor Loop Grid**, there's no built-in functionality to order products in a way that **out-of-stock items appear at the bottom of the list**.
This can be inconvenient for designers and users who prefer a coherent display of products where unavailable items are listed last.

The absence of this feature results in available products that have been added to the site long ago and are still in stock getting lost in the product listings, making it difficult for users to easily access them.

**If out-of-stock products are displayed at the bottom, available products are handy and greatly enhance the user experience.**

### Describe the solution you'd like.

I suggest implementing a feature in Elementor for WooCommerce that allows designers to sort products in a way that out-of-stock items are displayed at the bottom of the list. 

This can be achieved by introducing an option or setting within the Elementor interface that enables users to check an option to show out-of-stock products at the end, or not.
Or as another solution, you can consider several sorting criteria, for example, in order of loading date in the first step, and then in order of stock status.
Or any form that the specialists of the Elementor development team prefer :)

This enhancement would provide designers with greater flexibility and control over the layout of product listings, resulting in a more intuitive and user-friendly design experience.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by mohmmadebrahim** [comment]
Wonderful 👍
